### PR TITLE
Enable test for TensorFlow and CUDA 10

### PR DESCRIPTION
**Pull request by JanuszL**
- TensorFlow 1.13.1 supports CUDA 10 tests could be enabled for it

Signed-off-by: Janusz Lisiecki <jlisiecki@nvidia.com>

### CODE REVIEW

**Comment by JanuszL** [comment]
CUDA 9 690948
CUDA 10 690951

**Comment by JanuszL** [comment]
Rebuilds:
CUDA 9 #691103
CUDA 10 #691104

**Comment by JanuszL** [comment]
Rebuilds after rebase:
CUDA 9 #693472
CUDA 10 #693473

**Comment by JanuszL** [comment]
CUDA 9 #693829
CUDA 10 #693830

File: qa/setup_packages.py [hunk 1]
```diff
@@ -60,49 +70,49 @@ def print_install(variant, use, cuda):
...
         else:
             ret.append(key + "==" + val)
-        variant = variant // len(packages[key])
+        variant = variant // len(get_package(packages, key, cuda))
     # add all remaining used packages with default versions
     additional = [v for v in use if v not in packages.keys()]
     return " ".join(ret + additional)
 
-def print_remove(use):
+def print_remove(use, cuda):
```
Comment (Kh4L) [inline qa/setup_packages.py]: nit: I would change this function name to `get_remove_string` or `format_remove`

File: qa/setup_packages.py [hunk 2]
```diff
@@ -60,49 +70,49 @@ def print_install(variant, use, cuda):
...
+        print (cal_num_of_configs(args.use, args.cuda) - 1)
     elif args.remove:
-        print (print_remove(args.use))
+        print (print_remove(args.use, args.cuda))
     elif args.install >= 0:
-        if args.install > cal_num_of_configs(args.use):
+        if args.install > cal_num_of_configs(args.use, args.cuda):
             args.install = 1
-        print (print_install(args.install, args.use, "cu" + args.cuda))
+        print (print_install(args.install, args.use, args.cuda))
```
Comment (Kh4L) [inline qa/setup_packages.py]: Same for `print_install`

File: qa/setup_packages.py (hunk 1 above)
Comment (JanuszL) [reply qa/setup_packages.py]: Done

File: qa/setup_packages.py (hunk 2 above)
Comment (JanuszL) [reply qa/setup_packages.py]: Done

**Comment by JanuszL** [comment]
Rerunning #694399 and #694404

**Comment by JanuszL** [comment]
Rebuild 697821 and 697822

**Pull request merged**